lab1/Lexic.txt

```diff
@@ -0,0 +1,35 @@
+Alphabet:
+    1. Upper (A-Z) and lower case letters (a-z)
+    2. Underline character '_'
+    3. Decimal digits (0-9)
+
+Lexic:
+    1. Special symbols:
+        -operators: 
+            + - * / <- ++ and or mod smaller_than bigger_than equals not_equals
+        
+        -separators:
+            ( ) [ ] { } : ; space
+        
+        -reserved words:
+            let $ number string array_numbers if while for read print &START& &END&
+
+    2. Identifiers:
+        -it is a sequence and chars and digits, the first letter being a letter:
+            letter = "a" | "b" | ... | "z" | "A" | "B" | ... | "Z"
+            digit = "1" | "2" | ... | "9"
+            identifier = letter | letter {letter} {digit}
```

Comment:
Do you want that something like back2back to not be allowed as an identifier (i don't feel like that's what you want 
you should do letter {letter | digit }